nifti2/nifti_tool.c

```diff
@@ -3849,7 +3848,7 @@ int modify_field(void * basep, field_s * field, const char * data)
 
          case DT_INT8:
          {
-            max = 127;
+            max = UINT8_MAX;
```

Comment:
Is this change intentional? 127 vs. 255. Also 32767 vs. 65535 below.

Reply:
Whoops! Excellent catch.

Reply:
Fixed.